packages/utilities/psammead-styles/src/font-families.js

```diff
@@ -65,7 +65,7 @@ const helmetFontStyles = {
   `,
 };
 
-const nessimPersian = `font-family: NassimPersian, Arial, Verdana, Geneva, Helvetica, sans serif`;
+const nessimPersian = `font-family: NassimPersian, Arial, Verdana, Geneva, Helvetica, sans serif;`;
```

Comment:
```suggestion
const nassimPersian = `font-family: NassimPersian, Arial, Verdana, Geneva, Helvetica, sans serif;`;
```

Also can you check where else this is using the wrong spelling please

Reply:
👍 searched a repo and "nessim" only appears in this file - updated it to nassim